packages/web/src/pages/profile-page/ProfilePageProvider.tsx

```diff
@@ -259,9 +256,7 @@ class ProfilePage extends PureComponent<ProfilePageProps, ProfilePageState> {
     if (this.props.accountUserId) {
       this.props.updateCurrentUserFollows(true)
     }
-    if (this.props.relatedArtists && this.props.relatedArtists.length > 0) {
-      this.setState({ areArtistRecommendationsVisible: true })
-    }
+    this.setState({ areArtistRecommendationsVisible: true })
```

Comment:
this is where we could re-introduce the `SHOW_ARTIST_RECOMMENDATIONS_PERCENT` if we want